### PR TITLE
ACID404 Logo

### PR DESCRIPTION
**Pull request by PotluckProtocol**

### CODE REVIEW

File: mc.tokenlist.json [hunk 1]
```diff
@@ -2124,6 +2124,15 @@
       "tags": ["DeFi"],
       "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/joe-tokenlists/main/logos/0x239c1d2fb158113Dc4B10133b66d7516207879c5/logo.png"
     }
+    {
```
Comment (pecio222) [inline mc.tokenlist.json:2127]: Needs to separate with comma

File: mc.tokenlist.json [hunk 2]
```diff
@@ -2124,6 +2124,15 @@
...
+      "chainId": 43114,
+      "address": "0xF6C95C3A750CC7f6a8C96d9b08CC132A44c7Bd72",
+      "decimals": 18,
+      "name": "SICKO",
+      "symbol": "SICKO",
+      "tags": ["NFT"],
+      "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/joe-tokenlists/main/logos/0xf6c95c3a750cc7f6a8c96d9b08cc132a44c7bd72/logo.png"
+    },
```
Comment (pecio222) [inline mc.tokenlist.json:2135]: Remove this comma

File: logos/0xf6c95c3a750cc7f6a8c96d9b08cc132a44c7bd72/logo.png
Comment (pecio222) [inline logos/0xf6c95c3a750cc7f6a8c96d9b08cc132a44c7bd72/logo.png:1]: Address needs to be checksummed

File: mc.tokenlist.json [hunk 3]
```diff
@@ -2123,6 +2123,15 @@
       "symbol": "SKY",
       "tags": ["DeFi"],
       "logoURI": "https://raw.githubusercontent.com/traderjoe-xyz/joe-tokenlists/main/logos/0x239c1d2fb158113Dc4B10133b66d7516207879c5/logo.png"
+    },
+    {
+      "chainId": 43114,
+      "address": "0xF6C95C3A750CC7f6a8C96d9b08CC132A44c7Bd72",
+      "decimals": 18,
+      "name": "SICKO",
```
[the remark this answers is not in the document]
Comment (PotluckProtocol) [reply mc.tokenlist.json]: Ok, I re-submitted to have it match what is currently displaying when entering the address on TJ. Sorry about that!

**Comment by pecio222** [comment]
Folder name still not checksummed

**Comment by PotluckProtocol** [comment]
> Folder name still not checksummed

OOF sorry, for some reason GH desktop not recognizing the swap. Will get it fixed real quick, thanks for the patience!

**Comment by PotluckProtocol** [comment]
> > Folder name still not checksummed
> 
> OOF sorry, for some reason GH desktop not recognizing the swap. Will get it fixed real quick, thanks for the patience!

Updated

**Review by pecio222 — APPROVED** [review]

**Pull request merged**